tools-cli/src/test/java/com/hedera/hashgraph/client/cli/options/SubmitCommandTest.java

```diff
@@ -187,7 +183,7 @@ void submitOneTransactionWildCards_test() throws Exception {
 		final var wildcardName = FilenameUtils.getBaseName(transactionLocation).substring(0, 5) + "*";
 		logger.info("Submitting {}", wildcardName);
 		final String[] args =
-				{ "submit", "-t", TRANSACTIONS + File.separator + wildcardName, "-n", "INTEGRATION", "-o", RECEIPTS };
+				{ "submit", "-t", TRANSACTIONS + File.separator + wildcardName, "-n", NetworkEnum.TESTNET.getName(), "-o", RECEIPTS };
```

Comment:
Is this intended to be changed from `Integration` to `Testnet` ?

Reply:
Integration was giving me issues, for the most part, and I was told it might not be best to continue using the integration network. I changed it where possible. I created a TESTNET account, and credentials for that account are located in a .env file. Anyone wanting to test will need to do the same thing (create an account and their own .env file). I've not set an account up for the GitHub Action, to be honest. I overlooked that. I'll check that the unit tests pass in the action, and go from there.

Reply:
Sounds good. Thanks